rotkehlchen/chain/ethereum/defi/zerionsdk.py

```diff
@@ -125,6 +125,9 @@
     'Cometh • Tube',
     'Cometh • Staking',
     'Livepeer',
+    'Cozy',
+    'Cozy • Yearn',
+    'Cozy • Compound',
```

Comment:
This will also need frontend changes to add the icons in the frontend. An example commit doing it for the frontend is here: https://github.com/rotki/rotki/pull/2965/commits/0d01b80014a4b8f535116b6899db4ba074e61cb2